releasenotes/notes/0.19/sparse-pauli-internal-8226b4f57a61b982.yaml

```diff
@@ -6,7 +6,7 @@ upgrade:
     :class:`~qiskit.quantum_info.operators.PauliList`.
```

Comment:
I've rewritten this whole release note.  I've removed the upgrade note for `SparsePauliOp`, and expanded the deprecation notice to say more about the getter properties that you should now use.  I split the last line out into its own upgrade note, because that's actually what it is, and changed the warning emitted to a `FutureWarning`, because it's changing behaviour.

I'm a little worried about the speed that `PauliTable` is being completely stripped out of everything, and even deprecated itself.